src/helm.rs

```diff
@@ -18,6 +19,22 @@ pub struct HelmRelease {
     pub last_updated: String,
 }
 
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct HelmChart {
+    pub release_name: String,
+    pub name: String,
+    pub repo: HelmChartRepo,
+    pub version: String,
+    pub options: serde_yaml::Value,
```

Comment:
```suggestion
    pub values: serde_yaml::Value,
```

Reply:
We might as well be consistent with how Helm names it.

Reply:
Changed in df0076f.